esque/controller/topic_controller.py

```diff
@@ -93,6 +93,11 @@ def get_cluster_topic(self, topic_name: str) -> Topic:
         """Convenience function getting an existing topic based on topic_name"""
         return self.update_from_cluster(Topic(topic_name))
 
+    def get_local_topic(self, topic_name: str, get_from_cluster: bool = True) -> Topic:
+        if get_from_cluster:
```

Comment:
If you change the function like I mentioned above you don't need the flag here. :)